packages/python/plotly/plotly/io/_kaleido.py

```diff
@@ -260,4 +261,58 @@ def write_image(
         file.write(img_data)
 
 
-__all__ = ["to_image", "write_image", "scope"]
+def full_figure_for_development(fig, warn=True, as_dict=False):
+    """
+    Compute default values for all attributes not specified in the input figure and
+    returns the output as a "full" figure. This function calls Plotly.js via Kaleido
+    to populate unspecified attributes. This function is intended for interactive use
+    during development to learn more about how Plotly.js computes default values and is
+    not generally necessary or recommended for production use.
+
+    Parameters
+    ----------
+    fig:
+        Figure object or dict representing a figure
+
+    warn: bool
+        If False, suppress warnings about not using this in production.
+
+    as_dict: bool
+        If True, output is a dict with some keys that go.Figure can't parse.
+        If False, output is a go.Figure with unparseable keys skipped.
+
+    Returns
+    -------
+    plotly.graph_objects.Figure or dict
+        The full figure
+    """
+
+    # Raise informative error message if Kaleido is not installed
+    if scope is None:
+        raise ValueError(
+            """
+Image export using the "kaleido" engine requires the kaleido package,
```

Comment:
Should probably refer to full_figure_for_development instead of Image export here

Reply:
thanks! done